python/cog/server/http.py

```diff
@@ -36,6 +38,21 @@
 log = structlog.get_logger("cog.server.http")
 
 
+sentry_dsn=os.getenv("SENTRY_DSN", None)
+if sentry_dsn is not None:
+    sentry_sdk.init(
+        dsn=sentry_dsn,
+        # Set traces_sample_rate to 1.0 to capture 100%
+        # of transactions for performance monitoring.
+        traces_sample_rate=1.0,
+        # Set profiles_sample_rate to 1.0 to profile 100%
+        # of sampled transactions.
+        # We recommend adjusting this value in production.
+        profiles_sample_rate=1.0,
```

Comment:
In addition, it's better to use env variables to configure these rates, so we don't need to ask people to rebuild their image